ui/values/localizable/en.go

```diff
@@ -787,4 +787,5 @@ const EN = `
 "bondStrengthErrMsg" = "Bond Strength must be a valid number"
 "minimumBondStrength" = "Minimum Bond Strength is %d"
 "assets" = "Assets"
+"noWalletsAvailable" = "There are no wallets available for this function, Please try again later."
```

Comment:
telling them to try again later might not be of much help

we know the reason why they can't access the send page, so we should let them know

e.g. "You cannot spend from a watch only wallet, try creating another wallet"